daemon/CryptoImpl/cryptorequestprocessor.cpp

```diff
@@ -446,7 +454,12 @@ Daemon::ApiImpl::RequestProcessor::generateStoredKey_afterPreCheck(
             ikdRequest.setPluginName(keyTemplate.identifier().storagePluginName());
             ikdRequest.setOperation(Sailfish::Secrets::InteractionParameters::DeriveKey);
             ikdRequest.setAuthenticationPluginName(uiParams.authenticationPluginName());
```

Comment:
To me it is not immediately obvious what idk stands for. Also weird to name variable request, which is not request class.

Reply:
IKD is input key data - we generate a key from the key data via key derivation.  I can rename these to passphraseQuery or something instead.

Reply:
Renamed to promptParams